input/docs/handbook/view-models/index.md

```diff
@@ -1,167 +1,112 @@
 
 ### Common Mistakes and Misconceptions
 
-Many people believe that the MVVM pattern means that there should be zero code
-in the View code-behind, or that everything should be in XAML. While certain
-patterns such as Blend Triggers generally promote code reuse, this is
-generally an Antipattern. C# is a much more expressive, more concise language
-than XAML, and while it may be *possible* for you to create an entire complex
-View without writing any code, the result will be an unmaintainable, difficult
-to read mess.
+Many people believe that the MVVM pattern means that there should be zero code in the View code-behind, or that everything should be in XAML. While certain patterns such as Blend Triggers generally promote code reuse, this is generally an Antipattern. C# is a much more expressive, more concise language than XAML, and while it may be *possible* for you to create an entire complex View without writing any code, the result will be an unmaintainable, difficult to read mess.
 
-So then, how do I decide what to put in the View? Concepts such as scroll
-position and control focus are great examples of code that is View-specific.
-Handling animation and Window position / minimize are also great examples of
-code that often should be in the View.
+So then, how do I decide what to put in the View? Concepts such as scroll position and control focus are great examples of code that is View-specific. Handling animation and Window position / minimize are also great examples of code that often should be in the View.
 
-Another common misconception is that of separation - while it is very
-important that the ViewModel have no reference to the View or any of the
-controls that the View creates, **the reverse is not true**. The View is free
-to be very tightly bound to the ViewModel, and in fact, it is often useful for
-the View to "reach into" the ViewModel via `WhenAny` and `WhenAnyObservable`.
+Another common misconception is that of separation - while it is very important that the ViewModel have no reference to the View or any of the controls that the View creates, **the reverse is not true**. The View is free to be very tightly bound to the ViewModel, and in fact, it is often useful for the View to "reach into" the ViewModel via `WhenAny` and `WhenAnyObservable`.
 
-With the theory out of the way, let's see how to create ViewModels in
-ReactiveUI.
+With the theory out of the way, let's see how to create ViewModels in ReactiveUI.
 
-### Read-Write Properties
+# Creating a ViewModel class
 
-Properties that participate in change notification (i.e. that signal when they
-are changed), are written in the following way:
+**ReactiveObject** is the base object for ViewModel classes, and it implements `INotifyPropertyChanged`. In addition, ReactiveObject provides `Changing` and `Changed` Observables to monitor object changes. A typical ViewModel created using ReactiveUI framework should inherit from ReactiveObject:
```

Comment:
Worth mentioning `IReactiveObject`.  Base classes are great, but we should try and code to abstractions.  Most of the methods that matter go against `IReactiveObject` for that reason.